spark/src/test/java/com/netflix/iceberg/spark/source/TestFilteredScan.java

```diff
@@ -429,7 +429,7 @@ private File buildPartitionedTable(String desc, PartitionSpec spec, String udf,
     Table byId = TABLES.create(SCHEMA, spec, location.toString());
 
     // do not combine splits because the tests expect a split per partition
-    byId.updateProperties().set("read.split.target-size", "1").commit();
+    byId.updateProperties().set("read.split.target-size", "2048").commit();
```

Comment:
Can you update the comment to note that a lower value will also cause failures because the files are split? Something like "use a target size that will not combine or split files ..."